.sync/Files.yml

```diff
@@ -27,83 +27,35 @@
 #       set up to use the YAML file. The file is simply available for pipelines to be connected
 #       if needed.
 group:
-# Azure Pipelines - Custom - microsoft/mu_feature_config
-  - files:
-    - source: .sync/azure_pipelines/custom/mu_feature_config/Ubuntu-GCC5.yml
-      dest: .azurepipelines/Ubuntu-GCC5.yml
-      template: true
-    - source: .sync/azure_pipelines/custom/mu_feature_config/Windows-VS.yml
-      dest: .azurepipelines/Windows-VS.yml
-      template: true
-    repos: |
-      microsoft/mu_feature_config
-
-# Azure Pipelines - Custom - microsoft/mu_feature_ipmi
-  - files:
-    - source: .sync/azure_pipelines/custom/mu_feature_ipmi/Ubuntu-GCC5.yml
-      dest: .azurepipelines/Ubuntu-GCC5.yml
-      template: true
-    - source: .sync/azure_pipelines/custom/mu_feature_ipmi/Windows-VS.yml
-      dest: .azurepipelines/Windows-VS.yml
-      template: true
-    repos: |
-      microsoft/mu_feature_ipmi
-
-# Azure Pipelines - Custom - microsoft/mu_feature_mm_supv
-  - files:
-    - source: .sync/azure_pipelines/custom/mu_feature_mm_supv/Ubuntu-GCC5.yml
-      dest: .azurepipelines/Ubuntu-GCC5.yml
-      template: true
-    - source: .sync/azure_pipelines/custom/mu_feature_mm_supv/Windows-VS.yml
-      dest: .azurepipelines/Windows-VS.yml
-      template: true
-    repos: |
-      microsoft/mu_feature_mm_supv
-
-# Azure Pipelines - Custom - microsoft/mu_oem_sample
-  - files:
-    - source: .sync/azure_pipelines/custom/mu_oem_sample/Ubuntu-GCC5.yml
-      dest: .azurepipelines/Ubuntu-GCC5.yml
-      template: true
-    - source: .sync/azure_pipelines/custom/mu_oem_sample/Windows-VS.yml
-      dest: .azurepipelines/Windows-VS.yml
-      template: true
-    repos: |
-      microsoft/mu_oem_sample
-
-# Azure Pipelines - Matrix Dependent - All
+# Azure Pipelines - Matrix Dependent - CLANDPDB
   - files:

```

Comment:
Should we convert these files as well?

Reply:
Or should we remove them from this repo (and not do the sync)? We do not seem to run them anyway?

Reply:
Or delete them if they are not used. @makubacki , it looks like these were converted but I don't see any pipeline. was that intentional?

Reply:
They were added to bring existing support into mu_devops that was in mu_basecore. I agree that they're not used at the moment, and I'd suggest to delete them as well.

Reply:
Thanks, will remove them soon.

Reply:
The CLANGPDB files are gone.